bundle/tests/scorecard/kuttl/out.txt

```diff
@@ -0,0 +1,21 @@
+=== RUN   kuttl
```

Comment:
I believe this is produced by running kuttl locally. Probably should be added to gitignore. Not sure about the kubeconfig above